index.html

```diff
@@ -2322,8 +2322,8 @@ <h4>createDisclosureData</h4>
               </ol>
             </li>
             <li>
-Initialize |revealDocument| to the result of the "selectJsonLd"
-algorithm, passing |document|, and |combinedPointers| as |pointers|.
+Initialize |revealDocument| to the result of the algorithm in Section
```

Comment:
To match the wording elsewhere in this PR...
```suggestion
Initialize |revealDocument| to the result of the calling the algorithm in Section
```